electricitymap/contrib/capacity_parsers/REE.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime
+from logging import getLogger
+
+from requests import Response, Session
+
+from electricitymap.contrib.config import ZoneKey
+
+"""Disclaimer: Capacity for the Spanish isles is only avilable per archipelago."""
```

Comment:
We are just going to consider that all isles in one archipelago have the same capacity, isn't it?